root/etc/e-smith/templates/etc/nethserver/cockpit.allow/10base

```diff
@@ -0,0 +1,17 @@
+
+#
+# Allow groups to use cockpit 
+#
+
+{
```

Comment:
We can always list `root` and `domain admins` here. The file has to be checked by PAM only if `ShellOverrideStatus=enabled`.

Reply:
of course but you might have a chance that the template goes wrong

Reply:
What could go wrong with this template? `/etc/pam.d/cockpit` is a template too...

Reply:
ok